core/build.gradle.kts

```diff
@@ -36,6 +36,7 @@ kotlin {
                 api(libs.kotlinx.coroutines.core)
                 api(libs.uuid)
                 implementation(libs.tuulbox.collections)
+                implementation(libs.tuulbox.coroutines)
```

Comment:
Since none of the other platforms need this, this should be moved to `androidMain` dependencies.